library/Zend/Db/Sql/Predicate/Expression.php

```diff
@@ -19,7 +19,7 @@ class Expression extends BaseExpression implements PredicateInterface
      * @param string $expression
      * @param int|float|bool|string|array $valueParameter
      */
-    public function __construct($expression = null, $valueParameter = null /*[, $valueParameter, ... ]*/)
+    public function __construct($expression = null, $valueParameter = array() /*[, $valueParameter, ... ]*/)
```

Comment:
Wow, sorry, new baby, not a lot of sleep.  There is definitely an issue here though.  I'll take another look.